grammars/ruby.cson

```diff
@@ -1904,13 +1963,6 @@
     'match': '(?<=^|[ \\t])!|&&|\\|\\||\\^'
     'name': 'keyword.operator.logical.ruby'
   }
-  {
-    'comment': 'Safe navigation operator'
-    'match': '(&\\.)\\s*(?![A-Z])'
```

Comment:
It looks like this operator was removed?

Reply:
I put it in [line 2334](https://github.com/atom/language-ruby/pull/224/files#diff-e6c5c2e163c647d0b92203493f43f877R2334) since it was originally scoped as `punctuation.separator.method`.